aim/storage/hashing/c_hash.pyxbld

```diff
@@ -0,0 +1,5 @@
+from Cython.Build import cythonize
```

Comment:
@mahnerak  do we need this file to stay?

Reply:
Yes, this is needed for quick development. Makes it easier to use with pyximport.

Reply:
I mean the `c_hash` won't be changed a lot, could be understandable for `treeutils_`